requirements.txt

```diff
@@ -1,5 +1,5 @@
 # TODO: change this when next version of openwisp_users is released
-openwisp-users @ https://github.com/purhan/openwisp-users/tarball/master
+openwisp-users @ https://github.com/openwisp/openwisp-users/tarball/master
```

Comment:
@Purhan was there a reason for using your fork or was this a mistake? :smile:

Reply:
I believe it was a leftover